crayfish/metadata.txt

```diff
@@ -1,11 +1,11 @@
 ; the next section is mandatory
 [general]
 name=Crayfish
-qgisMinimumVersion=3.10
+qgisMinimumVersion=3.13
```

Comment:
with this change, we can also remove a few workarounds in the code. Check the `import` statements in the file headers, sometimes there is code like "if some class exists, do that and that" and this logic could be removed too.